### ISSUE TITLE
Draggable FlatList

### ISSUE THREAD

**Issue by shubhamkakkar** [galio-org/galio#117]
I was wondering if we could bring in some draggable list component, as this is the thing missing in most of the popular UI components library and devs have to use additional libraries!!

**Comment by palingheorghe** [comment]
What specifically do you have in mind when you say Draggable list component? I'm trying to visualize this on mobile and I'm not sure if this is really that used (I haven't seen it that much).

**Comment by AndreasEK** [comment]
The one place where I use that most is arranging playlists in Google Play Music (other music apps with playlists probably have that, too)

**Comment by palingheorghe** [comment]
Like a to-do app where you re-organize the items in whatever order you want them to be? @AndreasEK

**Comment by AndreasEK** [comment]
yes, that's how I understood the original requirement

**Comment by palingheorghe** [comment]
yeah, this is a good component for the future releases of Galio! I like the idea.

**Comment by amunrarara** [comment]
Bump this!
